python/sgl_jax/srt/layers/logits_processor.py

```diff
@@ -448,6 +448,7 @@ def _get_logits(
 
         logits = logits[:, : self.vocab_size] if logits.ndim > 1 else logits[: self.vocab_size]
 
-        if self.soft_cap:
+        if self.soft_cap and self.soft_cap > 0:
```

Comment:
static_validate_inputs checks whether soft_cap is greater than 0.0, so this seems unnecessary here, and during assignment, 0 is recognized and set to None.